README.md

```diff
@@ -77,3 +77,40 @@ To review:
 2. Go through the code.
 3. If something is not clear, ask for clarification. If someone has already asked just thumbs up that comment.
 4. Add an approving comment saying "Good work!". If someone has already commented thumbs up on that.
+
+## How To Contribute and Run Software
+* Fork This Repo by clicking on button as shown in image.
+![Fork](screenshots/1.png)
```

Comment:
if you have added screenshots to the gitignore then no images will be there as reference to.

Reply:
so should I remove screenshots? or we should keep it ?